insights/combiners/tomcat_virtual_dir_context.py

```diff
@@ -0,0 +1,39 @@
+"""
+TomcatVirtualDirContextCombined - check VirtualDirContext option used in Tomcat
+===============================================================================
+
+This combiner provides information about whether a VirtualDirContext is used in config files in
+both default locations or location derived from running Tomcat process.
+
+Examples::
+
+    >>> shared[TomcatVirtualDirContextCombined].data
+    {'/usr/share/tomcat/conf/server.xml':
+     ['    <Resources className="org.apache.naming.resources.VirtualDirContext"'],
+     }
+"""
+from ..core.plugins import combiner
+from ..parsers.tomcat_virtual_dir_context import TomcatVirtualDirContextFallback, \
+    TomcatVirtualDirContextTargeted
+
+
+@combiner(requires=[[TomcatVirtualDirContextFallback, TomcatVirtualDirContextTargeted]])
```

Comment:
@skontar the decorator changes for 3.x, you'll need to change this to:
`@combiner([TomcatVirtualDirContextFallback, TomcatVirtualDirContextTargeted])`

Take a look a some of the other combiners for more examples.  They have all been converted to the 3.x format.